tests/unit/fmt-test.js

```diff
@@ -16,7 +17,11 @@ test('\'%@08 %@07 %@06 %@05 %@04 %@03 %@02 %@01\'.fmt(\'One\', \'Two\', \'Three\
 });
 
 test('\'data: %@\'.fmt({ id: 3 }) => \'data: {id: 3}\'', function(assert) {
-  assert.equal(fmt('data: %@', [{ id: 3 }]), 'data: {id: 3}');
+  // The whitespace output of hashes passed through Ember.inspect differs from 3.x. We need have different output whether it's 2.x or 3.x
+  let emberVersion = /\d/.exec(VERSION);
+  let emberInt = parseInt(emberVersion[0], 10);
+  let hashOutput = emberInt > 2 ? '{ id: 3 }' : '{id: 3}'
+  assert.equal(fmt('data: %@', [{ id: 3 }]), `data: ${hashOutput}`);
```

Comment:
When running tests through ember-try, i.e. using Ember 2.x and 3.x, we need to tweak the output of the test depending what version of Ember we are running.

Ember 2.x: `{id: 3}`
Ember 3.x: `{ id: 3 }`